### ISSUE TITLE
Removing an error has side-effects

### ISSUE THREAD

**Issue by MichaelVdheeren** [emberjs/data#3529]
Use case:

* Textfield in IE8 and IE9 where value is bound to a value of the model
* Validations triggered on focus-out and added to the errors list of the model

Problem:

* When removing the error the attribute is apparently modified (https://github.com/emberjs/data/blob/e89732a5aefb6a81b46927da1c6f048f4aede85e/packages/ember-data/lib/system/model/errors.js#L291)
* This triggers calculations in our code, by which attributes of the input change
* In IE8 and IE9 this causes a re-render of the input field and thus a focus-out resulting in a loop. In any other browser the behavior is ok.

Question:

Why is removing an error different from clear, clear does not trigger notifying the attributes of the model, remove does, why?

**Comment by tchak** [comment]
Could you post part of your template where this is happening? I see no reason why this is happening. The `notifyPropertyChange` call you showing is related to attribute on the `errors` object not on the model itself.

**Comment by MichaelVdheeren** [comment]
Hey, we've found out what the problem is. Sorry for leaving this open. It has no relation to Ember Data whatsoever (again sorry). When the errors we're set, we displayed them on our template. So when the error was added, the height of the element in which the errors we're shown was changed. That same div contained our input element.

Now we use a resize event to force a [repaint](http://stackoverflow.com/questions/8840580/force-dom-redraw-refresh-on-chrome-mac) for an iPad problem. But that specific code causes a focus-out from the input element, only in IE.

Bizare, but we couldn't expect any different from IE I guess.
